src/commands/deploy.ts

```diff
@@ -181,10 +179,22 @@ export default class Deploy extends Command {
         config.region === 'iran' ? '.iran.liara.run' : '.liara.run';
       const urlLogMessage = DEV_MODE
         ? // tslint:disable-next-line: no-http-string
-          `    ${chalk.cyan(`http://${config.app}.liara.localhost`)}`
-        : `    ${chalk.cyan(`https://${config.app}${defaultSubdomain}`)}`;
+          `    ${`http://${config.app}.liara.localhost`}`
+        : `    ${`https://${config.app}${defaultSubdomain}`}`;
       this.log(urlLogMessage);
 
+      const { domains } = await this.got(
+        `v1/domains?project=${config.app}`
+      ).json<IGetDomainsResponse>();
+
+      for (const domain of domains) {
+        if (domain.certificatesStatus === 'ACTIVE') {
+          this.log(chalk.white(`    https://${domain.name}`));
+        } else {
+          this.log(chalk.white(`    http://${domain.name}`));
+        }
```

Comment:
What do you think of extracting the protocol part to a new variable called `const protocol`.  Then you can get rid of this if and else statement.

Reply:
a good idea.

Reply:
i fixed it.